functions/go/upsert-resource/main.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/GoogleContainerTools/kpt-functions-catalog/functions/go/upsert-resource/generated"
+	"github.com/GoogleContainerTools/kpt-functions-catalog/functions/go/upsert-resource/upsertresource"
+	"github.com/ghodss/yaml"
```

Comment:
This is no longer actively maintained.
You can use k8s' permanent fork of it: sigs.k8s.io/yaml

Reply:
Removed.